docker-compose.yml

```diff
@@ -1,9 +1,59 @@
+version: "3"
+
 services:
   db:
-    image: postgres
-    ports:
-      - 5432:5432
+    container_name: db
+    image: postgres:alpine
     environment:
       - POSTGRES_USER=app
       - POSTGRES_PASSWORD=app
-      - POSTGRES_DB=db
\ No newline at end of file
+      - POSTGRES_DB=db
+    networks:
+      - my_network
+  
+  db-push:
```

Comment:
Make sure the database schema is synchronized with prisma schema (only run once, close after push)